.github/paths-labeller.yml

```diff
@@ -0,0 +1,53 @@
+---
+- "beats":
+  - "detection_rules/beats.py"
+  - "etc/beats_schemas/**/*"
+- "documentation":
+  - "./**/*.md"
+- "Domain: Cloud":
+  - "rules/integrations/aws/**/*.toml"
+  - "rules/integrations/azure/**/*.toml"
+  - "rules/integrations/gcp/**/*.toml"
+  - "rules/integrations/google-workspace/**/*.toml"
+  - "rules/integrations/microsoft-365/**/*.toml"
+  - "rules/integrations/okta/**/*.toml"
+- "Domain: Endpoint":
+  - "rules/windows/**/*.toml"
+  - "rules/linux/**/*.toml"
+  - "rules/macos/**/*.toml"
+- "ecs":
+  - "detection_rules/ecs.py"
+  - "etc/ecs_schemas/**/*"
+- "ML":
+  - "rules/ml/**/*"
+- "OS: Linux":
+  - "rules/linux/**/*.toml"
+- "OS: macOS":
+  - "rules/macos/**/*.toml"
+- "OS: Windows":
+  - "rules/windows/**/*.toml"
+- "Platform: AWS":
+  - "rules/integrations/aws/**/*.toml"
+- "Platform: Azure":
+  - "rules/integrations/azure/**/*.toml"
+- "Platform: Crowdstrike":
+  - "rules/integrations/crowdstrike/**/*.toml"
+- "Platform: CyberArkPas":
+  - "rules/integrations/cyberarkpas/**/*.toml"
+- "Platform: GCP":
+  - "rules/integrations/gcp/**/*.toml"
+- "Platform: Google Workspace":
+  - "rules/integrations/google-workspace/**/*.toml"
+- "Platform: Microsoft 365":
+  - "rules/integrations/microsoft-365/**/*.toml"
+- "Platform: Okta":
+  - "rules/integrations/okta/**/*.toml"
+- "python":
+  - "detection_rules/**/*.py"
+- "RTA":
+  - "rta/**/*"
+- "Rule: Deprecation":
+  - "rules/_deprecated/**/*"
+# tests
+- "test-suite":
```

Comment:
I think we can drop this one.
I was thinking of this as more like meta tests or a test exclusive PR.
Any new functionality that adds a corresponding test doesn't make sense to label with this in my opinion